lib/plugins.ts

```diff
@@ -1,7 +1,15 @@
-var webdriver = require('selenium-webdriver');
-import {Logger} from './logger';
 import * as Q from 'q';
+import * as webdriver from 'selenium-webdriver';
+
 import {ConfigParser} from './configParser';
+import {Logger} from './logger';
+
+declare module 'q' {
```

Comment:
It's a deprecated and undocumented method that Protractor still uses. It can be difficult to get the reviewers to accept such a PR as they usually want to see a link to API docs. So I'm not sure it'd be the right thing to do.